contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -3477,6 +3477,11 @@ pltsql_store_func_default_positions(ObjectAddress address, List *parameters, con
 
 	if (HeapTupleIsValid(oldtup))
 	{
+		if(original_query && strncasecmp("alter", original_query, 5) == 0) {
```

Comment:
Is it possible the original_sql = " alter xxx" , ( starts with a space or \r\n )

Reply:
No it gets parsed so even if there are spaces before `ALTER` original_sql will not have those spaces / extra characters in front